changelog.txt

```diff
@@ -18,7 +18,7 @@ Added
 
 Changed
 +++++++
-- Command line interface for ``exec`` changed parameter name from ``jobid`` to ``job_id`` (#363).
+- Command line interface always uses ``--job-id`` instead of ``--jobid`` (#363, #386).
```

Comment:
The previous changelog message for #363 was not meaningful -- the user-facing API did not change. This PR does change the user-facing API for some CLI tasks. I decided to have only one changelog line that summarizes the user-facing changes here.